src/frequenz/client/base/_grpchacks.py

```diff
@@ -0,0 +1,46 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Hacks to deal with multiple grpc libraries.
```

Comment:
I think it is fine to mention this is a hack in the documentation. Though I personally don't like to have the word `hacks` in the module name. What do you think about renaming the module from `_grpchacks.py` to something around `_grpc_utils.py`? (I didn't suggest `_grpc_base_exceptions.py` as I understood it'd be too specific to the current state and not flexible for extending it)

Reply:
I intentionally used  `hacks` to make it look dirty. I considered `utils` but it is too broad, and it could invite people to add random stuff there. I picked hacks because it entitles that it is not something desirable to have, and that hopefully it is temporary. If you had a bad reaction to it, I think my naming was successful :laughing:

Reply:
It did trigger some noise in my mind :D 
Hopefully it won't stay forever :crossed_fingers: